pkg/controller/certificatesigningrequest/ipsec_csr_approver.go

```diff
@@ -123,7 +134,7 @@ func (ic *ipsecCSRApprover) verifyCertificateRequest(req *x509.CertificateReques
 }
 
 func (ic *ipsecCSRApprover) verifyIdentity(nodeName string, csr *certificatesv1.CertificateSigningRequest) error {
-	if csr.Spec.Username != antreaAgentServiceAccountName {
+	if csr.Spec.Username != getAntreaAgentServiceAccount() {
```

Comment:
Even though I don't think that `verifyIdentity` is called very often, we may want to just add `antreaAgentServiceAccountName` as a field to the `ipsecCSRApprover` struct, and initialize the field by calling `getAntreaAgentServiceAccount()` when we instantiate `ipsecCSRApprover` objects. It may be a good idea to have a `newIPsecCSRApprover` helper function to instantiate.